app/controllers/candidate_interface/prefill_application_form_controller.rb

```diff
@@ -37,5 +40,17 @@ def prefill_candidate_application_form
     def prefill_application_or_not_params
       params.fetch(:candidate_interface_prefill_application_or_not_form, {}).permit(:prefill)
     end
+
+    def destroy_blank_application
+      blank_application.destroy if blank_application?
+    end
+
+    def blank_application?
+      blank_application.created_at == blank_application.updated_at
+    end
+
+    def blank_application
+      @blank_application ||= current_candidate.application_forms.first
+    end
```

Comment:
@benswannack - I've added a guard clause to ensure we only destroy applications that are actually blank. What do you think?

Not sure if this logic should sit here or somewhere else though ... Could potentially create a service but is that overkill? 🤔

Reply:
Maybe `#blank_application?` could move to the model 🤔

Reply:
There's already a `#blank_application?` method on `ApplicationForm`

Reply:
Ah yes - there is! I'll fix